sass/components/_status-icon.scss

```diff
@@ -208,3 +208,14 @@
         fill: rgba(var(--center-channel-color-rgb), 0.56);
     }
 }
+
+.app__body {
+    .status {
+        .status--away {
```

Comment:
should these selectors be moved to https://github.com/mattermost/mattermost-webapp/pull/6721/files#diff-be141d211e13ecd6a1975debc4929fdeR63 ?

Reply:
The link you provided takes me to the diff of this PR. Is there a different link you meant to provide?

Reply:
Hmm is .app__body selector not needed in some form? Or is it implicit? I'm still wrapping my head around the codebase.

Reply:
No worries, yeah don't think that would be required in that case :).

Reply:
Hindsight's 2020 because I think I tested incorrectly, but when I'm testing with the change the theme button doesn't in the sidebar doesn't match if I change the away theme color
<img width="568" alt="Screen Shot 2020-10-14 at 11 52 16 AM" src="https://user-images.githubusercontent.com/11154862/96032657-fbc0d280-0e2c-11eb-8f66-401ec5595cd4.png">
<img width="221" alt="Screen Shot 2020-10-14 at 11 52 21 AM" src="https://user-images.githubusercontent.com/11154862/96032663-ff545980-0e2c-11eb-9b97-016f8d156ee2.png">

Reply:
o. So with this new change your test above isn't working @rohanrk?

Reply:
actually do you mind trying it here instead: https://github.com/mattermost/mattermost-webapp/blob/3f3c2b8ae44f65030c2f8c0649534cca9299ee14/sass/components/_status-icon.scss#L142

Reply:
Looks like the same issue occurs when switching it over to that line @ali-farooq0

Reply:
@rohanrk
Oh my. Well that's unfortunate. Alright i guess maybe it _does_ need the `app__body` parent. Apologies for the churn on this. Feel free to revert and test to make sure all is well and that should be good for now.

Reply:
Done! Tested and it looks good on my end. Let me know if you want me to squash the commits.